client/state/posts/selectors.js

```diff
@@ -359,6 +360,46 @@ export const isEditedPostDirty = createSelector(
 );
 
 /**
+  * Returns true if the edited post has content
+  * (title, excerpt or content not empty)
+  *
+  * @param  {Object}  state  Global state tree
+  * @param  {Number}  siteId Site ID
+  * @param  {Number}  postId Post ID
+  * @return {Boolean}        Whether the edited post has content or not
+ */
+export const editedPostHasContent = createSelector(
```

Comment:
Could this selector be simplified to:

```js
const editedPost = getEditedPost( siteId, postId );
return (
	some( [ 'title', 'excerpt' ], ( field ) => !! editedPost[ field ].trim() ) ||
	! isPostContentEmpty( editedPost.content )
);
```

Notably leveraging `getEditedPost` instead of recreating that behavior.